proximity/ProximitySensor.https.html

```diff
@@ -5,10 +5,26 @@
 <link rel="help" href="https://w3c.github.io/proximity/">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/generic-sensor/resources/generic-sensor-helpers.js"></script>
 <script src="/generic-sensor/generic-sensor-tests.js"></script>
-<div id="log"></div>
+
 <script>
 
-runGenericSensorTests('ProximitySensor');
+'use strict';
+
+const kReadings = {
+    readings: [
+        [1.12345, 2.12345, true]
```

Comment:
This is supposed to be a raw reading value, so a boolean does not make much sense here. For proper testing, you should provide a number that the sensor code will convert into a boolean.

Reply:
ProximitySensor is not supported in mojo yet, so I think we could optimize this in future. If you agree on this, I will add a TODO note. :)

Reply:
Do you mean it's not handled in `resources/chromium/generic_sensor_mocks.js`? If that's the case, it should be pretty easy to add the required bits there in this PR to make it complete. If not, can you clarify what you're referring to, and how this has been tested so far?

Reply:
Done.